semanticdb/metacp/src/main/scala/scala/meta/internal/javacp/Javacp.scala

```diff
@@ -151,9 +157,13 @@ object Javacp {
           m.descriptor == method.descriptor
         }
         val suffix =
-          if (synonyms.length == 1) ""
+          if (synonyms.lengthCompare(1) == 0) ""
```

Comment:
I think this is an overkill, but I don't think this is a blocker.

Reply:
I wouldn't say this is idiomatic, regardless of what Intellij thinks about it. `lengthCompare` is likely to be faster than `length` on big lists, but I don't think that performance gains are worth the readability hit here.

Reply:
I like using a method designed exactly for this usecase. The readability hit is miniscule it doesn't bother me much.